downstream/modules/platform/proc-controller-user-tokens.adoc

```diff
@@ -0,0 +1,35 @@
+[id="proc-controller-user-tokens"]
+
+= Users - Tokens
+
+The *Tokens* tab is only present for your user (yourself). 
+Before you add a token for your user, you may want to link:https://docs.ansible.com/automation-controller/latest/html/userguide/applications_auth.html#ug-applications-auth-create[Create an application] if you want to
+associate your token with it. 
+
+You can also create a _Personal Access Token_(PAT) without associating it with any application. 
+
+To create a token for your user:
+
+.Procedure
+. Select your user from the *Users* list view to configure your OAuth 2 tokens.
+. Select the *Tokens* tab from your user's profile.
+. Click btn:[Add] to open the *Create Token* window.
+. Enter the following information:
+* *Application*: Enter the name of the application with which you want to associate your token. 
++
+Alternatively, you can search for it by clicking the image:search.png[Search,15,15] icon. 
```

Comment:
In the editorial review I was advised to follow the procedures section of IBM SG when there was more than one option:
https://www.ibm.com/docs/en/ibm-style?topic=format-procedures

This procedure would start like:

" 1. Create a token for your user by using one of these methods:
       *Select your user...
       
I'm not sure how best to structure this now though.

Reply:
There aren't two methods, there are two options for *Application* either add the name, or search for it.